TAT.homework6/src/transport/Bicycle.java

```diff
@@ -0,0 +1,29 @@
+package transport;
+
+/**
+ * Represents bicycle.
+ * @author Yury Suponev
+ */
+public class Bicycle extends MuscleMovementWay {

```

Comment:
this class is quite useless as it doesn't do anything, just store some fields. It means you need to rework your class hierarchy